src/common/LoadingBanner.js

```diff
@@ -7,7 +7,7 @@ import { View } from 'react-native';
 import { useSelector } from '../react-redux';
 import { getLoading } from '../selectors';
 // eslint-disable-next-line import/no-useless-path-segments
```

Comment:
This line can probably disappear too, right?

Reply:
Yep, thanks!